spec/models/mailing_list_spec.rb

```diff
@@ -515,4 +530,12 @@ def create_subscription(subscriber, excluded = false, *role_types)
     sub
   end
 
+  def subscription_tags(names)
+    tags = names.map do |name|
+      ActsAsTaggableOn::Tag.create_or_find_by!(name: name)
+    end
+    tags.map do |tag|
+      SubscriptionTag.create!(tag: tag)
```

Comment:
this creates a SubscriptionTag entry without a subscription_id ... use new instead ?
if you're going to make subscription_id not null, this won't work anyway :wink: